fsharp-backend/tests/testfiles/int.tests

```diff
@@ -168,4 +168,20 @@ Int.divide_v0 1 0 = Test.typeError_v0 "Division by zero"
 (List.range_v0 1 5) |> List.map_v0 (fun x -> Int.random_v1 20 10) |> List.map_v0 (fun x -> (x >= 10) && (x <= 20)) = [true; true; true; true; true]
 
 Int.sum_v0 [1;2] = 3
-Int.sum_v0 [1.0;2] = Test.typeError_v0 "Expected the argument `a` to be a list of ints, but it was `[ \n  1., 2\n]`"
\ No newline at end of file
+Int.sum_v0 [1.0;2] = Test.typeError_v0 "Expected the argument `a` to be a list of ints, but it was `[ \n  1., 2\n]`"
+
+Int.parse_v0 "" = Error "Expected to parse string with only numbers, instead got \"\""
+Int.parse_v0 "-00001" = Ok -1
+Int.parse_v0 "-10001" = Ok -10001
+Int.parse_v0 "-4611686018427387903" = Ok -4611686018427387903L
+Int.parse_v0 "-4611686018427387904" = Ok -4611686018427387904L
```

Comment:
Can you add some tests with whitespace around the number? (front, back, between `-` and the number).

What happens with `+` in the number, can we test that too?

Maybe a test for other representations that might be accidentally parsed without our knowledge, to confirm they aren't supported (0x, 0b, 5E-5)?

What about a number in octal format (078)?